can/io/asc.py

```diff
@@ -47,14 +46,19 @@ def __init__(
         :param base: Select the base(hex or dec) of id and data.
                      If the header of the asc file contains base information,
                      this value will be overwritten. Default "hex".
+        :param relative: Select whether the timestamps are `relative` (starting
+                     at 0.0) of `absolute` (starting at the system time).
+                     Default `True = relative`.
         """
         super().__init__(file, mode="r")
 
         if not self.file:
             raise ValueError("The given file cannot be None")
         self.base = base
         self._converted_base = self._check_base(base)
+        self.relative_timestamp = relative
```

Comment:
I'd also name the argument `relative_timestamp` to keep it consistent with the attribute.